client/src/components/card.tsx

```diff
@@ -1,8 +1,21 @@
 import { useState, useEffect, useRef } from 'react';
 import { Link, useNavigate } from 'react-router-dom';
 import axios from 'axios';
+import Modal from './modal'
+import { IAppointment } from '../../../server/interfaces/appointment.interface';
+interface IProps extends IAppointment {
+  _id?: string;
+  setShowModal?: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
dont think we need this anymore with the routing direction we're going. same goes for `setValue` below.